engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/ReplTest.scala

```diff
@@ -239,7 +239,7 @@ class ReplTest extends InterpreterTest with BeforeAndAfter with EitherValues {
       }
       eval(code)
       val errorMsg =
-        "Unexpected type provided for argument `that` in Number.+"
+        "Unexpected type provided for argument `that` in Int_64.+"
```

Comment:
`Int_64` is the name we want for the fixed-width 64-bit integers, so this should be `SmallInteger`.